std/socket.d

```diff
@@ -3522,8 +3522,8 @@ class UdpSocket: Socket
             @property @trusted Address localAddress() { assert(0); }
             @trusted ptrdiff_t send(const(void)[] buf, SocketFlags flags) { assert(0); }
             @safe ptrdiff_t send(const(void)[] buf) { assert(0); }
-            @trusted ptrdiff_t sendTo(const(void)[] buf, SocketFlags flags, Address to) { assert(0); }
-            @safe ptrdiff_t sendTo(const(void)[] buf, Address to) { assert(0); }
+            @trusted ptrdiff_t sendTo(const(void)[] buf, SocketFlags flags, in Address to) { assert(0); }
+            @safe ptrdiff_t sendTo(const(void)[] buf, in Address to) { assert(0); }
```

Comment:
@CyberShadow my thoughts exactly.  But maybe our const overloads could just cast away the const?
Presuming no one has overloaded sendTo and actually expected non const.